### PR TITLE
Bdu 89/tx details logs

### PR DESCRIPTION
**Pull request by ryuash**
## Description

This PR will add the log component to Transaction details

## Checklist
- [ ] Ran Linting
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [ ] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,3 +21,4 @@
 - Add Stake Weight Component ([\#41](https://github.com/forbole/big-dipper-2.0-solana/issues/41))
 - Fix: update general chain config report issue url ([\#98](https://github.com/forbole/big-dipper-2.0-solana/issues/98))
 - Fix AvatarName if name === '""' or length is empty return address ([\#95](https://github.com/forbole/big-dipper-2.0-solana/issues/95))
+- Add tx details logs component
```
Comment (yayay927) [inline CHANGELOG.md:24]: Please add url of issue in changelog.

Comment (ryuash) [reply CHANGELOG.md:24]: There is no issue for this though

Comment (yayay927) [reply CHANGELOG.md:24]: I see, then no need to add~

Comment (ryuash) [reply CHANGELOG.md:24]: You need to approve my PR

**Pull request merged**